packages/cli/generators/openapi/templates/src/controllers/controller-template.ts.ejs

```diff
@@ -19,8 +23,12 @@ imports.forEach(i => {
  */
 <%- decoration %>
 export class <%- className %> {
-  constructor() {}
-
+  <% if(true){ %>
+    constructor(@inject('services.<%- serviceClassName %>')
+    protected <%- serviceClassNameCamelCase %>: <%- serviceClassName %>) {}
```

Comment:
This is only adding service proxy injector. It will also need basic crud method implementation too. Otherwise, this inject will not help.

Reply:
The methods are already added but they throw the error `Not implemented`.  I've updated the controller implementation [here](https://github.com/loopbackio/loopback-next/pull/9188/files#diff-072f490a9a63e0dd68f8fd1cdf8f34429440b1548a76a92d22f3757451c5c362R352).

Reply:
ok